src/components/dropbox-backup/DropboxBackup.jsx

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React, {useState} from 'react';
+import {Dropbox} from 'dropbox';
+import './dropbox-backup.scss';
+
+type Props = {
+  notes: Array<Object>,
+};
+
+const getAccessToken = () => {
+  const accessToken = sessionStorage.getItem('dropbox_access_token');
```

Comment:
Using sessionStorage for this because it's assumed everything in localStorage is a note.